### ISSUE TITLE
Allow filter feature in Log Stream to work for non-production logs

### ISSUE THREAD

**Issue by lem-hatch** [statsig-io/statsig-feedback#46]
I would like to filter my non-production logs in the Log Stream by any of the displayed data. Event and User ID are the most helpful.

**Comment by tore-statsig** [comment]
Yeah this makes sense as a feature request, but the backend for each of those logstreams is sufficiently different that this isnt as easy as it might seem.  I've raised it with the team as a feature request but I don't expect this to be a quick turn around
